src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -335,14 +343,26 @@ bool CDROM_Interface_Ioctl::ReadSectorsHost([[maybe_unused]] void *buffer,
 	return false; /*TODO*/
 }
 
-void CDROM_Interface_Ioctl::InitAudio(const int device_number)
+void CDROM_Interface_Ioctl::InitAudio()
 {
 	if (mixer_channel) {
 		return;
 	}
 
-	std::string name = std::string(ChannelName::CdAudio) + "_" +
-	                   std::to_string(device_number);
+	bool found_unique_name = false;
+	std::string name;
+	for (int i = 0; i < 32; ++i) {
+		name = std::string(ChannelName::CdAudio) + "_" + std::to_string(i);
```

Comment:
`const auto MaxCdAudioChannel = 32`

Plus it would be nice to include a comment explaining that we'll create `CDAUDIO_1`, `CDAUDIO_2` for every mounted CD-ROM image/drive or something.

Reply:
Fixed